tests/constants/label.constants.ts

```diff
@@ -68,6 +38,15 @@ export const COLOR_RESPONSE_CANNOT_REPRODUCE = 'ffebcc';
 export const COLOR_STATUS_DONE = 'a6a6a6';
 export const COLOR_STATUS_INCOMPLETE = '000000';
 
+// Label definition constants
+const DEFINITION_SEVERITY_LOW = LABEL_DEFINITIONS.severityLow;
+const DEFINITION_SEVERITY_MEDIUM = LABEL_DEFINITIONS.severityMedium;
+const DEFINITION_SEVERITY_HIGH = LABEL_DEFINITIONS.severityHigh;
+const DEFINITION_FUNCTIONALITY_BUG = LABEL_DEFINITIONS.typeFunctionalityBug;
+const DEFINITION_DOCUMENTATION_BUG = LABEL_DEFINITIONS.typeDocumentationBug;
+const DEFINITION_ACCEPTED = LABEL_DEFINITIONS.responseAccepted;
+const DEFINITION_REJECTED = LABEL_DEFINITIONS.responseRejected;
```

Comment:
We can make use of this PR to just replace all the current constants with `LABEL_DEFINITIONS.xx` also. 

In that case, we wouldn't another PR to solve this issue. @ducbinh2611

Reply:
Yes, I agree with @seanlowjk